multidim-interop/impl/js/v0.46/package.json

```
@@ -1,19 +1,19 @@
 {
   "dependencies": {
-    "@libp2p/multidim-interop": "1.0.0-c4eff4c5"
+    "@libp2p/multidim-interop": "1.0.1"
```

Maybe use a range here?

```suggestion
    "@libp2p/multidim-interop": "^1.0.1"
```

---

It's fixed specifically for purposes of strict reproducibility, see https://github.com/libp2p/test-plans/pull/288 for more details.

---

Is the lockfile not enough for this?

---

What would be the benefit of adding the range?

---

So you don't have to manually update patch release versions all the time.  The lockfile gives you repeatability in CI, the range gives you less to do in dev.